openapi.yaml

```diff
@@ -16,175 +16,148 @@ host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/poker-api
 produces:
   - application/json
 paths:
-  /pets:
-    get:
-      summary: List all pets
+  /games:
+    post:
```

Comment:
Currently, you only have a method for your API: `POST /games`, is it correct?

Reply:
I added more methods. Does that look good?